src/ocean/util/log/Logger.d

```diff
@@ -956,3 +967,36 @@ unittest
     // The first entry in the array
     test!("==")(Log.convert("trace", Level.Error), Level.Trace);
 }
+
+// Test that argumentless format call does not shrink the output
+unittest
+{
+    static class Buffer : Appender
+    {
+        public struct Event { Logger.Level level; cstring message; }
+        public Event[] result;
+
+        public override Mask mask () { return 42; }
+        public override cstring name () { return "BufferAppender"; }
+        public override void append (LogEvent e)
+        {
+            this.result ~= Event(e.level, e.toString());
+        }
+    }
```

Comment:
This will (in the future) be refactored in a more visible place so it'll be accessible for users to use (so you can test the output of your loggers in UT). But since it's a patch release, I didn't want to introduce new symbols.